### PR TITLE
completed implementation

### PR DESCRIPTION
**Pull request by alikouhestani**

### CODE REVIEW

File: app/views/DashboardView.scala.html [hunk 1]
```diff
@@ -21,10 +21,28 @@
...
+ </div>
+ <div class="govuk-notification-banner__content">
+  <p class="govuk-notification-banner__heading">
+   @messages("dashboard.inactive.message")
+   <a class="govuk-notification-banner__link" href="https://www.gov.uk/government/consultations/draft-guidance-multinational-top-up-tax-and-domestic-top-up-tax" rel="noreferrer noopener" target="_blank">@messages("dashboard.inactive.link")</a>.
+  </p>
+ </div>
+</div>
+}
+
```
Comment (AKKUM) [inline app/views/DashboardView.scala.html:45]: Do we have any hmrc notification banner library? did you check?

File: test/controllers/DashboardControllerSpec.scala [hunk 2]
```diff
@@ -187,8 +188,8 @@ class DashboardControllerSpec extends SpecBase with ModelGenerators {
         status(result) mustEqual SEE_OTHER
         redirectLocation(result) mustBe Some(controllers.routes.JourneyRecoveryController.onPageLoad().url)
       }
-    }
 
+    }
   }
 
 }
```
Comment (AKKUM) [inline test/controllers/DashboardControllerSpec.scala:195]: do we need any test scenario for inactive

File: app/views/DashboardView.scala.html (hunk 1 above)
Comment (alikouhestani) [reply app/views/DashboardView.scala.html:45]: We do but it still uses html if you need to include a hyper link in there so I didn't see the point
https://design-system.service.gov.uk/components/notification-banner/

File: test/controllers/DashboardControllerSpec.scala (hunk 2 above)
Comment (alikouhestani) [reply test/controllers/DashboardControllerSpec.scala:195]: I would say we should have a test case ideally but the implementation of it was very tricky and given that it was priority and I only had a couple of hours to work on it, I only relied on regression test alone.

File: app/views/DashboardView.scala.html (hunk 1 above)
Comment (AKKUM) [reply app/views/DashboardView.scala.html:45]: Better to use the hmrc Lib if available. for accessibility point of view as well as any basic changes happens on future library upgrade pick automaticall. But I am happy to approve.

**Review by AKKUM — APPROVED** [review]

File: app/views/DashboardView.scala.html (hunk 1 above)
Comment (alikouhestani) [reply app/views/DashboardView.scala.html:45]: That is a html libraries but its just written in html as opposed to twirl. or is that not what you mean?

**Pull request merged**